### ISSUE TITLE
[Feature] Allow users to choose whether to prioritize webp or not

### ISSUE THREAD

**Issue by viliml** [manga-download/hakuneko#6158]
Some websites like bato.to provide their images in webp format when the `Accept` header contains the `image/webp` value, and in png format otherwise, even if the `Accept` header contains the values `image/*` or `*/*` meaning that webp is accepted.
One way to get png images is to add the following snippet to https://github.com/manga-download/hakuneko/blob/66877d7694cc8d2fc90271610ff2fafbf2d3553e/src/web/mjs/engine/Request.mjs#L362:
```js
win.webContents.session.webRequest.onBeforeSendHeaders((details, callback) => {
  details.requestHeaders['Accept'] = 'text/html,application/xhtml+xml,application/xml;q=0.9,image/*,*/*;q=0.8';
  callback({ requestHeaders: details.requestHeaders });
});
```
The reason why I didn't open a pull request is because I'm not confident I can nicely add it as a configuration setting so that users who prefer webp can retain the previous behavior.

**Comment by MikeZeDev** [comment]
Too much a change to be done for every website reliably.

We already support format selection for some websites. If you feel like a website should got such an option just open a ticket for said website.

Other devs opinion may vary.

**Comment by Sheepux** [comment]
I agree, this is a per connector option as every website may chose another machanism to give alternative format

**Comment by MikeZeDev** [comment]
On the other end, there can be a global option. Since most websites doesnt provide different format we dont have to abide by the option .

In any case, i dont see that coming for hakuneko, too much work. That could be something for haruneko i guess.
